### PR TITLE
adding placeholder and license info

### PR DESCRIPTION
**Pull request by donNewtonAlpha**

### CODE REVIEW

**Comment by donNewtonAlpha** [comment]
This pr add license information, adds placeholders for directories and adds the vendor directory with dependencies

File: vendor/github.com/golang/snappy/AUTHORS [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# This is the official list of Snappy-Go authors for copyright purposes.
```
Comment (SeanCondon) [inline vendor/github.com/golang/snappy/AUTHORS:1]: I think all of this vendor directory can be added to git ignore - see `onos-topo` to a good example of how we maintain and build Go modules

**Review by SeanCondon — COMMENTED** [review]
@donNewtonAlpha - I also created a branch protection rule for this repo

**Comment by donNewtonAlpha** [comment]
closing to remove the vendor dependencies

**Pull request closed without merging**